### ISSUE TITLE
Can't seem to properly indent ? : statements

### ISSUE THREAD

**Issue by 0x80** [yannickcr/eslint-plugin-react#625]
In our codebase we use quite a bit of this type of if/else rendering:

``` javascript
      {
          condition
          ? <div>
              <div> something </div>
            </div>
          : <div>
              <div> something </div>
            </div>
        }
```

Whatever indentation I try I always seem to get a jsx-indent error. Is there a way to format this without the linter complaining?

**Comment by yannickcr** [comment]
Seems related to #540

**Comment by jrajav** [comment]
#540 has been closed - did it fix this?

**Comment by ljharb** [comment]
```jsx
{condition ? (
  <div>
    <div> something </div>
  </div>
) : (
  <div>
    <div> something </div>
  </div>
)}
```

or
```jsx
{condition
  ? (
    <div>
      <div> something </div>
    </div>
  )
  : (
    <div>
      <div> something </div>
    </div>
  )
}
```
?

**Comment by jrajav** [comment]
Neither -

```
{condition
  ? <div>
      <div> something </div>
    </div>
  : <div>
      <div> something </div>
    </div>
}

**Comment by ljharb** [comment]
Right, I'm suggesting two forms that should satisfy the rule.

If those work, and your example doesn't, then it seems like an issue when parens are omitted.

**Comment by IanVS** [comment]
I would also love to be able to use:

```
condition
? <div>
     <div> something </div>
  </div>
: <div />
```

rather than 

```
condition
? <div>
    <div> something </div>
</div>
: <div />
```

The closing </div> is aligned to the opening tags, rather than the `?` and `:`.

Would this be possible to add as an option?

**Comment by jrajav** [comment]
Will this ever be fixed? We've had to simply leave JSX indents unchecked for a long time now because this rule doesn't address the way we (and many others) indent ternaries.

**Comment by jrajav** [comment]
Is this ever going to get fixed? To this day, still have to keep jsx-indent turned off because this doesn't work as expected (or at least have an option).

**Comment by jrajav** [comment]
Made a new issue since @ljharb perhaps indicated it's an issue with lacking parens (but that seems like it should be a valid use case?).

**Comment by ljharb** [comment]
To clarify, I think it should be a supported use case, but I don't think it's a good style :-)

**Comment by jrajav** [comment]
@ljharb Thanks for the clarification! That's good to hear. In your opinion should it be the default or an added option?

**Comment by ljharb** [comment]
I think that the default should be identical when parens are omitted or not - iow, delete the parens out of https://github.com/yannickcr/eslint-plugin-react/issues/625#issuecomment-253309000 for the style without them.

The more compact style requested here should be an option.
